### DISCUSSION TITLE
Im not able to show contentitems button for admin users

### DISCUSSION THREAD

**Discussion by ufoloko** [2sic/2sxc#3012] [General] [answered]
Im trying to show 2 buttons, one with new action and one with contentitems action, but contentitems action is not being displayed to admin users, just to superusers, what i need to do ?

**Comment by iJungleboy** [comment, marked as answer]
check out https://docs.2sxc.org/basics/edit/edit-ux/toolbars/button-ui.html#show

**Reply by ufoloko** [reply]
Thanks for the response, I have checked documentation sent but I still don't get it. My code is 

@Edit.Toolbar(toolbar: new object[] {   
    new { 
        command = new { 
            action = "new", 
            contentType = "Category",
            prefill = new { ModuleId = Dnn.Module.ModuleID }
        } 
    },
    new {
        command = new { 
            action = "contentitems", 
            contentType = "Category", 
            filters = new { ModuleId = Dnn.Module.ModuleID } 
        }
    }
}, settings: new { hover="left" })

How i should add suggested option show=true into my code ? button for new action is being showed but contentitems action is not being showed. Maybe some additional permission is needed ?

Thanks

**Reply by iJungleboy** [reply]
You seem to be using the very old api https://docs.2sxc.org/js-code/toolbars/old.html which is not recommended any more. 

It works, but it was really hard to use 🤷🏽‍♂️. I believe it was ca. like this:

```
command = new {
action = "new",
contentType = "Category",
prefill = new { ModuleId = Dnn.Module.ModuleID },
show = true
}
```

But I highly urge you to go for the newer 10.27+ API https://docs.2sxc.org/js-code/toolbars/simple.html

Or even the best, latest v14+ API using the @Kit.Toolbar service https://docs.2sxc.org/api/dot-net/ToSic.Sxc.Services.IToolbarService.html and https://docs.2sxc.org/api/dot-net/ToSic.Sxc.Edit.Toolbar.IToolbarBuilder.html

You can see all in action with live source code here: https://2sxc.org/dnn-tutorials/en/razor/ui100/page

You would then probably need something like

```
@Kit.Toolbar.Empty(Content)
   // https://docs.2sxc.org/api/dot-net/ToSic.Sxc.Edit.Toolbar.IToolbarBuilder.html#ToSic_Sxc_Edit_Toolbar_IToolbarBuilder_New_System_Object_System_String_System_Object_System_Object_System_Object_System_String_
  .New("Category", prefill: new { ModuleId = CmsContext.Module.Id} ) 

// https://docs.2sxc.org/net-code/services/toolbar-builder/data-filter.html
 .Data("Category", filter: new { ModuleId = CmsContext.Module.Id}, ui: "show=true" ) ;
```

**Reply by ufoloko** [reply]
Thanks a lot, i already solved with the 10.27+ api, i´m not able yet to upgrade to v14+.

I really appreciate the time you take to help.
